<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartTracking</class>
 <widget class="QDialog" name="StartTracking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txtSrcFile">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnParcourirIn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Parcourir</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Depuis fichier</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbCamera">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Depuis caméra</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbIndexCamera">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtOutFile">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="txtParcourirOut">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Parcourir</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>361</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QCheckBox" name="cbShowHSV">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher HSV</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cvReduceNoise">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string> Appliquer réduction de bruit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cvShowTH">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher Threeshold</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnExecuter">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Executer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuitter">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
